docs/DeveloperGuide.md

```diff
@@ -7,32 +8,127 @@
+|reserve|reserve {package_id},{contact_name},{contact_number},{number_pax} <br/> e.g reserve 3,John,91234567,3|
+|remove|remove {package_id},{contact_number} <br/> e.g remove 1,8888888 <br/> remove an existing reservation|
+|reservations|reservations {package_number} <br/> eg. reservations 2 <br/> print all reservations for a given travelPackageID|
+
+### Test Case 1
+
+New employees can enter `packages` to view all available packages by the agency.
+
+Additionally, they can enter `info {package number}` to view more in-depth information about the package.
+
+![image](https://user-images.githubusercontent.com/64303732/161110520-8ed18ddc-b356-43d5-95ed-7df5ca2de91d.png)
+
+Employees can also input new reservations using `reserve {package_id},{contact_name},{contact_number},{number_pax}`
+
+They can then view and verify reservations for that package using `reservation {package_number}`
+
+![image](https://user-images.githubusercontent.com/64303732/161111482-a2f9600d-d1f0-45b1-9a51-ef28848e4b57.png)
+
+![image](https://user-images.githubusercontent.com/64303732/161111551-c1acfd1e-90f3-426e-b8e3-175f050dba42.png)
+
+
+## Feature - Help Command 
+Aim: Displays a list of all available commands that the user can refer to as a guide
+
+![image](https://user-images.githubusercontent.com/64303732/161110894-f9226a6c-e9f2-4cf3-9158-72cc8df19604.png)
+
+
+
+## Feature - Storage
+#### Initialisation (Loading Data)
+The sequence diagram that shows how `Storage` is created and the data is loaded from the saved files when the program is initialised is shown below: </br>
+![](LoadFileSeqDiag.png)
+
+1. `TARBS` creates a Storage object with the relevant file name (`filePath`).
+2. `TARBS` then calls the `createPackages()` method of the Storage class
+3. `storage` will call the `parseSavedFile()` method, which initiates a while loop that iterates through the text file and processes the string data using the respective `parse` methods.
+4. A new `TravelPackage` object is constructed with the relevant data.
+5. A new `Packages` object is created using the `TravelPackage` object and returned back to `TARBS`.
+
+#### Saving data
+The sequence diagram that shows how `Storage` is used to save the current list of travel packages and reservations is shown below: </br>
+
+![](SaveFilesSeqDiag.png)
+
+1. `TARBS` calls the `savePackageToFile()` method of `storage`, passing in the `packages` object.
+2. `storage` will then use a loop to iteratively retrieve the data in string format from the individual `TravelPackage` objects.
+3. `storage` will create a new `FileWriter` object to write the string data into a text file.
+
+## Feature - Parser
+The sequence diagram shows an example of how a user input is parsed and returns a new `Command`.
+
+For simplicity, `ByeCommand` is shown here:
+
+![image](https://user-images.githubusercontent.com/53790951/160289728-5a17b2ea-0599-48f4-a18b-cadfbbcbfb11.png)
```

Comment:
1. Consider moving the condition for the [alt] path to beside the icon, instead of on the first method call. 
2. Consider adding crosses to indicate EoL if applicable
3. The classes are lacking a colon
4. Is there any code after return ByeCommand? If not consider ending both activation bars on the return arrow.